src/server/sv_main.cpp

```diff
@@ -1041,6 +1045,60 @@ qboolean SV_CheckPaused( void ) {
 	return qtrue;
 }
 
+void SV_CheckCvars(void) {
```

Comment:
Ok so client's snapshotMsec depends on sv_maxRate, sv_MinRate, sv_enforceSnaps, sv_fps and client's "snaps" userinfo. Please put all these calculations in a single function that will be used here and in UserInfoChanged. Remove wishsnaps data member. Doesn't matter if it's slow in SV_CheckVars(), it is called very rarely.